addons/im_livechat/models/discuss_channel.py

```diff
@@ -510,9 +510,14 @@ def _close_livechat_session(self, **kwargs):
                 body=Markup('<div class="o_mail_notification o_hide_author">%s</div>')
                 % self._get_visitor_leave_message(**kwargs),
                 message_type='notification',
-                subtype_xmlid='mail.mt_comment'
+                subtype_xmlid='mail.mt_comment',
+                silent=self._silent_message_on_close_livechat(),
             )
 
+    def _silent_message_on_close_livechat(self):
+        # No need to notify the operator with the `visitor has left channel` message if the operator isn't a human.
+        return bool(self.chatbot_current_step_id)
```

Comment:
This message post is more generic than what this comment makes it look like.

No assumption about operator can be made here. Anyone could have decided to watch this channel without joining it.

I think it's important to keep this message in all cases, it's not something we want to remove.

Reply:
The comment still applies, this change is not ok.
`silent` is not about _who_ receives the notification, but about the notification itself.

Please revert this change, and open a PR for the fix you are attempting to make with this with a clear reason why + way to reproduce + a test, so we can discuss about it without the rest of the diff.